cmd/gendoc/main.go

```diff
@@ -337,7 +339,8 @@ func parseDoc(method, doc string) *DocBlock {
 		doc = strings.TrimLeft(doc, method)
 	}
 	doc = strings.TrimLeft(doc, " ")
-
+	// generates the link to the sample archive
+	doc = reSampleArchive.ReplaceAllString(doc, "[$0](./$1)")
```

Comment:
This is a very simple new feature that looks for the sample data provided in the documentation and converts it into a link, making it easier to access the sample file from the documentation.